mrrc/npm.py

```diff
@@ -213,10 +216,10 @@ def gen_package_meatadata_file(version_metadata: NPMVersionMetadata, root='/'):
     version_dict[version_metadata.get_version()] = version_metadata.__dict__
     package_metadata.set_versions(version_dict)
 
-    logging(package_metadata.__str__())
+    logger.debug(package_metadata.__str__())
```

Comment:
We shouldn't be calling __str__ function just like the __init__ function it's called by default when the object is converted to string.

Reply:
Removed.